CMakeLists.txt

```diff
@@ -1084,6 +1130,35 @@ if(BUILD_PYTHON)
   endif()
 endif()
 
+if(NOT DUCKDB_EXPLICIT_PLATFORM)
+  set(VERSION_SOURCES tools/utils/test_platform.cpp)
+
+  add_executable(duckdb_platform_binary ${VERSION_SOURCES})
+  link_threads(duckdb_platform_binary)
+
+  set_target_properties(duckdb_platform_binary PROPERTIES OUTPUT_NAME duckdb_platform_binary)
+  set_target_properties(duckdb_platform_binary PROPERTIES RUNTIME_OUTPUT_DIRECTORY
+                                         ${PROJECT_BINARY_DIR})
+  if (WIN32)
+    add_custom_target(
```

Comment:
Maybe I'm missing something but this if/else seems to not change anything?

Reply:
Right, nice catch. This has been refactored away, now using properly CMake to detect the binary (instead of building the name on the spot). I will take this out.

Reply:
Handled in follow up commit, thanks